infrastructures/docker/docker-client/src/test/java/org/eclipse/che/infrastructure/docker/client/DockerConnectorTest.java

```diff
@@ -1341,6 +1332,34 @@ public void shouldThrowIOExceptionWhenBuildImageButNoSuccessMessageInResponse()
     verify(dockerResponse).getInputStream();
   }
 
+  @Test(
+    expectedExceptions = DockerException.class,
+    expectedExceptionsMessageRegExp = "Docker image build exceed timeout.*"
```

Comment:
Would it better to rewrite it in the following way "Docker image build exceed timeout .* seconds." ?